variables.tf

```
@@ -91,6 +91,17 @@ variable "admins" {
   description = "A list of administrators for the instance crypto units. See [instructions](https://github.com/terraform-ibm-modules/terraform-ibm-hpcs#before-you-begin) to create administrator signature keys. You can set up to 8 administrators. Required if auto_initialization_using_recovery_crypto_units set to true. "
 }
 
+variable "base64_encoded_admins" {
+  type = list(object({
+    name  = string # max length: 30 chars
+    key   = string #  base64 encoded value of signature key files if key files are created using TKE CLI and are not using a third-party signing service
+    token = string # sensitive: the administrator password/token to authorize and access the corresponding signature key file
+  }))
+  default     = []
+  sensitive   = true
+  description = "A list of administrators for the instance crypto units. See [instructions](https://github.com/terraform-ibm-modules/terraform-ibm-hpcs#before-you-begin) to create administrator signature keys. You can set up to 8 administrators. Required if auto_initialization_using_recovery_crypto_units set to true. The signature keys should passed as base64 encoded values."
```

I think we should assume that descriptions are plain text, not markdown. I think that not all places handle MD correctly.

I shortened a bit and moved the readme stuff to the end, but I think more context is needed. How is this different from the `admins` variable. Are they mutually exclusive? Do users need to know which to use?  

```suggestion
  description = "A list of up to 8 administrators for the instance crypto units. Required if auto_initialization_using_recovery_crypto_units is set to true. Pass the signature keys as base64 encoded values. For information about administrator signature keys, see the readme file."
```

---

Yea, they are mutually exclusive. Only one can be passed at a time.